### PR TITLE
update nexo importer to support new export format

### LINKED ISSUE #4502: Import Data > Nexo
## Import Data > Nexo

Error Message in Red > Invalid CSV format, missing required field: 'Currency'

### System Description

Operating system: MacOS 12.3.1
Rotki version: 1.24.1

### PR DESCRIPTION
**Pull request by prettyirrelevant**
Closes #4502

### CODE REVIEW

File: rotkehlchen/tests/utils/dataimport.py [hunk 1]
```diff
@@ -697,133 +698,110 @@ def assert_nexo_results(rotki: Rotkehlchen):
     warnings = rotki.msg_aggregator.consume_warnings()
     errors = rotki.msg_aggregator.consume_errors()
     assert len(errors) == 0
-    assert len(warnings) == 0
+    assert len(warnings) == 6
```
Comment (nebolax) [inline rotkehlchen/tests/utils/dataimport.py:701]: What are these 6 warnings about? I would adjust our logic to have 0 warnings or at least check the warnings' content that we get here, what do you think?

File: rotkehlchen/tests/utils/dataimport.py [hunk 2]
```diff
@@ -697,133 +698,110 @@ def assert_nexo_results(rotki: Rotkehlchen):
...
```
Comment (nebolax) [inline rotkehlchen/tests/utils/dataimport.py:702]: And one more question: does the new csv file check all the cases? (I mean cases when something is off)

Comment (prettyirrelevant) [reply rotkehlchen/tests/utils/dataimport.py:702]: Nexo simply changed the column headers to `Amount, Currency` -> `Input Amount, Input Currency, Output Amount, Output Currency`.

File: rotkehlchen/tests/utils/dataimport.py (hunk 1 above)
Comment (nebolax) [reply rotkehlchen/tests/utils/dataimport.py:701]: Does this mean these transactions are completely ignored? Can we adjust to support them?

Comment (prettyirrelevant) [reply rotkehlchen/tests/utils/dataimport.py:701]: The entries are completely ignored, yeah.
https://github.com/rotki/rotki/blob/develop/rotkehlchen/data_import/importers/nexo.py#L42

Comment (nebolax) [reply rotkehlchen/tests/utils/dataimport.py:701]: Seems like `ignored_entries` are not the case, are they? If transaction type is mentioned in `ignored_entries`, it is skipped and no warning is raised, or am I misunderstanding something?

Comment (prettyirrelevant) [reply rotkehlchen/tests/utils/dataimport.py:701]: Yeah, you're right. I was the one mixing things together.

**Review by LefterisJP — APPROVED** [review]
LGTM. Just needs a rebase. I will do it myself since otherwise looks good and we can merge fast.

**Pull request merged**